servers/account-data-deleter/src/batchDeleteHandler.ts

```diff
@@ -35,10 +38,15 @@ export class BatchDeleteHandler {
    * poll events
    * @param pollOnInit whether to start polling when the class is
    * instantiated, primarily for testing (default=true);
+   * @param unleashClient optional unleash client, intended
+   * to use mock for testing. Otherwise will pull in the globally
+   * initialized unleash instance. Can consider DI here and elsewhere
```

Comment:
have we reached that point in the ts code???!

Reply:
haha well, I think it's more of a style preference thing. we have mangled constructor "DI" like this courtesy of me elsewhere :joy: